frontend/.eslintrc.typescript.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    parserOptions: {
+        project: ['tsconfig.json', 'jsconfig.json'],
+        tsconfigRootDir: __dirname,
+    },
+    rules: {
+        '@typescript-eslint/no-floating-promises': ['error'],
+    },
+};
```

Comment:
Please import and spread the original configuration here:

```suggestion
import eslintrc from `./.eslintrc.js`;

module.exports = {
    ...eslintrc,
    parserOptions: {
        project: ['tsconfig.json', 'jsconfig.json'],
        tsconfigRootDir: __dirname,
    },
    rules: {
        ...eslintrc.rules,
        '@typescript-eslint/no-floating-promises': ['error'],
    },
};
```

Reply:
Right now `eslint` uses the default `.eslintrc.js` and this file as an "extra" config, so this would just duplicate the config. I could do this and add the `no-eslintrc` option to the command which lets eslint ignore the default file.

Reply:
> and this file as an "extra" config

Interesting, but how does this "extra" logic handles new rules? Are they merged together with the default ones or do they overwrite them? I expect that they will overwrite, that's why I think it's required to merge them manually, otherwise default rules wouldn't be checked in the CI. I would anyway would make it explicit by accepting the suggestion (or what's the downside of applying them "twice"?)

Reply:
They do get merged together actually, but I agree that there's probably no downside to simply applying them twice, and for readability it makes sense to spread the original config here so that it's easier to understand that this file is just the normal config + some extra stuff.